### PR TITLE
Update delete-an-inactive-mailbox.md

### PR DESCRIPTION
**Pull request by kellybos**
Inactive mailboxes are retained for 183 days after a hold is removed. The contents in this article need to make this difference between soft-deleted mailboxes (that were never Inactive) clear. Confirmed with Le-Wu Tung that this content should be updated and triaged by appropriate PM team.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/delete-an-inactive-mailbox.md [hunk 1]
```diff
@@ -201,25 +201,25 @@ If the In-Place Hold contains a large number of source mailboxes, it's possible
...
+- **An inactive mailbox is a type of soft-deleted mailbox.** In Exchange Online, a soft-deleted mailbox is a mailbox that's been deleted but can be recovered within a specific retention period. A previously inactive mailbox will be available as a soft-deleted mailbox in Exchange Online for 183 days. This means that the mailbox can be recovered within 183 days of being soft-deleted. After 183 days, a soft-deleted mailbox is marked for permanent deletion and can't be recovered.
 
 - **What happens after you remove the hold on an inactive mailbox?** The mailbox is treated like other soft-deleted mailboxes and is marked for permanent deletion after the 30-day soft-deleted mailbox retention period expires. This retention period starts on the date when the mailbox was first made inactive. This date is known as the soft-deleted date, which is the date the corresponding user account was deleted or when the Exchange Online mailbox was deleted with the **Remove-Mailbox** cmdlet. The soft-deleted date isn't the date on which you remove the hold.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/delete-an-inactive-mailbox.md]:
```suggestion
- **What happens after you remove the hold on an inactive mailbox?** The mailbox is treated like other soft-deleted mailboxes and is marked for permanent deletion after the 183-day soft-deleted mailbox retention period expires. This retention period starts on the date when the mailbox was first made inactive. This date is known as the soft-deleted date, which is the date the corresponding user account was deleted or when the Exchange Online mailbox was deleted with the **Remove-Mailbox** cmdlet. The soft-deleted date isn't the date on which you remove the hold.
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/delete-an-inactive-mailbox.md:206]: Please confirm.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@kellybos

**Comment by yogkumgit** [comment]
Gentle reminder @kellybos Please review and update with feedback

File: microsoft-365/compliance/delete-an-inactive-mailbox.md (hunk 1 above)
Comment (kellybos) [reply microsoft-365/compliance/delete-an-inactive-mailbox.md]: Update looks good. Thank you!

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@kellybos - could you please commit my changes? Thanks.

**Comment by kellybos** [comment]
Added comment above. Changes look good. Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by markjjo** [comment]
@kellybos PR merged. Thanks for making this change. Sorry for the delay.

**Pull request merged**